docs/source/guides/using_dask_entitysets.rst

```diff
@@ -25,16 +25,17 @@ For this example, we will create a very small pandas dataframe and then convert
     dask_df
 
 
-Now that we have our Dask dataframe, we can start to create the ``EntitySet``. The current implementation does not support variable type inference for Dask entities, so we must pass a dictionary of variable types using the ``variable_types`` parameter when calling ``es.entity_from_dataframe()``. Aside from needing to supply the variable types, the rest of the process of creating an ``EntitySet`` is the same as if we were using pandas dataframes.
+Now that we have our Dask dataframe, we can start to create the ``EntitySet``. The current implementation does not support variable type inference for Dask entities, so we must pass a dictionary of variable types using the ``variable_types`` parameter when calling ``es.add_dataframe()``. Aside from needing to supply the variable types, the rest of the process of creating an ``EntitySet`` is the same as if we were using pandas dataframes.
```

Comment:
This should also refer to logical_types instead of variable_types, but we can also update later during our big documentation update efforts.

Reply:
updated